app/utils/general.js

```diff
@@ -3,6 +3,17 @@
 
 import {Alert} from 'react-native';
 
+export function fromAutoResponder(post) {
+    return Boolean(post.type && (post.type === 'system_auto_response'));
```

Comment:
Not sure if this one is in the redux lib constants or not, but should probably add it.